sources/packages/backend/libs/sims-db/src/entities/notification-permanent-failure-error.type.ts

```diff
@@ -0,0 +1,4 @@
+export interface PermanentFailureError {
```

Comment:
maybe u can add a comment that its the format from GC notify, that's the reason we are not following our enum standards here